.eslintrc.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Guessing `.json` to get syntax highlighting and linting? I had a google around for side-effects, but all seems fine.